projects/rero/ng-core/src/lib/record/record-statistics/record-statistics.component.spec.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Is this test is really necessary ? Previously you really operate some checks. Now this test seems empty and could be removed.